src/main/java/com/johan/blignaut/entelect/vitalityincubator/helpers/GeoDBMappingHelper.java

```diff
@@ -21,18 +22,24 @@ public class GeoDBMappingHelper {
     private final GeodbApi geodbApi;
     private final GeoDBMappingService geoDBMappingService;
 
+    @Value("${itemsperpage}")
+    private int itemsPerPage = 7;
```

Comment:
PLease remove this assignment to show that the value is being pulled from the configuration correctly.